glusterd2/brick/validation.go

```diff
@@ -19,12 +19,43 @@ const (
 )
 
 // InitChecks is a set of checks to be run on a brick
+//if bool value is  ture --> dont do any check
 type InitChecks struct {
-	IsInUse  bool
-	IsMount  bool
-	IsOnRoot bool
+	IsInUse          bool
+	IsMount          bool
+	IsOnRoot         bool
+	IsCreateBrickDir bool
 }
 
+// PrepareChecks initializes InitChecks based on req
+func PrepareChecks(force bool, req map[string]bool) *InitChecks {
+	c := &InitChecks{}
+
+	if !force {
```

Comment:
This works. However, the code readability can be improved as follows:

```go

c := &InitChecks{}

if req.Force {
    // skip all checks if force is set to true
    return c
}

// do all the checks except the ones explicitly excluded 

c.IsInUse = true
...

```

Reply:
done